src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -85,6 +86,46 @@ public PersistentDataStore() {
     return users;
   }
 
+  /**
+   * Loads all User objects who are admins from the Datastore service and returns them in a List.
+   *
+   * @throws PersistentDataStoreException if an error was detected during the load from the
+   *     Datastore service
+   */
+  public List<User> loadAdmins() throws PersistentDataStoreException {
+
+    List<User> admins = new ArrayList<>();
+
+    // Retrieve all users from the datastore.
+    Query query = new Query("chat-users");
+    PreparedQuery results = datastore.prepare(query);
+
+    for (Entity entity : results.asIterable()) {
+      try {
+        String userName = (String) entity.getProperty("username");
+        String password = (String) entity.getProperty("password");
+        Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
+        String about = (String) entity.getProperty("about");
+        // changing to is_admin gives null pointer exception?
+        boolean isAdmin = (Boolean) entity.getProperty("isAdmin");
```

Comment:
Ah, don't worry about it then. Probably an issue with AppEngine and I'd rather you spend time coding then fiddling with that.